### PR TITLE
update several sql statement docs

### PR DESCRIPTION
**Pull request by bb7133**
Update:

- sql-statements/sql-statement-drop-sequence.md
- sql-statements/sql-statement-show-character-set.md
- sql-statements/sql-statement-show-collation.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
https://github.com/pingcap/docs-cn/pull/3294
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-drop-sequence.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ The `DROP SEQUENCE` statement drops the sequence object in TiDB.
```
Comment (yikeke) [inline sql-statements/sql-statement-drop-sequence.md:27]: line 18-21 needs to be deleted according to https://github.com/pingcap/docs-cn/pull/3294/files: 

```
**OptTemporary:**
![OptTemporary](/media/sqlgram/OptTemporary.png)

```

File: sql-statements/sql-statement-show-character-set.md [hunk 2]
```diff
@@ -11,17 +11,9 @@ This statement provides a static list of available character sets in TiDB. The o
...
 
-![ShowStmt](/media/sqlgram/ShowStmt.png)
-
-**ShowTargetFilterable:**
-
-![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
-
-**CharsetKw:**
-
-![CharsetKw](/media/sqlgram/CharsetKw.png)
```
Comment (yikeke) [inline sql-statements/sql-statement-show-character-set.md:24]: These three lines should not be deleted according to https://github.com/pingcap/docs-cn/pull/3294/files.

File: sql-statements/sql-statement-show-collation.md [hunk 3]
```diff
@@ -9,252 +9,57 @@ aliases: ['/docs/dev/reference/sql/statements/show-collation/']
 
 This statement provides a static list of collations, and is included to provide compatibility with MySQL client libraries.
 
-> **Note:** TiDB currently only supports binary collations.
+> **Note:** 
+>
+> Results of `SHOW COLLATION` changes with the fact whether the "new collation framework" is enabled, detailed information of "new collation framework" can be found in [Character Set and Collation](/character-set-and-collation.md))
```
Comment (yikeke) [inline sql-statements/sql-statement-show-collation.md:14]:
```suggestion
> Results of `SHOW COLLATION` vary when the "new collation framework" is enabled. For new collation framework details, refer to [Character Set and Collation](/character-set-and-collation.md).
```

File: sql-statements/sql-statement-show-character-set.md [hunk 4]
```diff
@@ -41,8 +33,9 @@ mysql> SHOW CHARACTER SET;
 
 ## MySQL compatibility
 
-This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
+The usage of this statement is understood to be fully compatible with MySQL. However, charsets in TiDB may have different default collations compared with MySQL, the details can be found in [Compatibility with MySQL](/mysql-compatibility.md). Any other compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
```
Comment (yikeke) [inline sql-statements/sql-statement-show-character-set.md:36]:
```suggestion
The usage of this statement is understood to be fully compatible with MySQL. However, charsets in TiDB may have different default collations compared with MySQL. For details, refer to [Compatibility with MySQL](/mysql-compatibility.md). Any other compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
```

File: sql-statements/sql-statement-show-collation.md [hunk 5]
```diff
@@ -9,252 +9,57 @@ aliases: ['/docs/dev/reference/sql/statements/show-collation/']
...
+| utf8_general_ci    | utf8    |   33 |         | Yes      |       1 |
+| utf8mb4_general_ci | utf8    |   45 |         | Yes      |       1 |
++--------------------+---------+------+---------+----------+---------+
+7 rows in set (0.02 sec)
 ```
 
 ## MySQL compatibility
 
-TiDB does not support collations other than binary. This statement is only provided to ensure compatibility with MySQL.
+The usage of this statement is understood to be fully compatible with MySQL. However, charsets in TiDB may have different default collations compared with MySQL, the details can be found in [Compatibility with MySQL](/mysql-compatibility.md). Any other compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
```
Comment (yikeke) [inline sql-statements/sql-statement-show-collation.md:60]:
```suggestion
The usage of this statement is understood to be fully compatible with MySQL. However, charsets in TiDB may have different default collations compared with MySQL. For details, refer to [Compatibility with MySQL](/mysql-compatibility.md). Any other compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
```

File: sql-statements/sql-statement-drop-sequence.md (hunk 1 above)
Comment (bb7133) [reply sql-statements/sql-statement-drop-sequence.md:23]: Thanks for the detailed reviews!

**Pull request merged**